docs/zh_cn/某商业列存DB和CarbonData查询性能对比.md

```diff
@@ -0,0 +1,109 @@
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one or more 
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership. 
+    The ASF licenses this file to you under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with 
+    the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+    
+    Unless required by applicable law or agreed to in writing, software 
+    distributed under the License is distributed on an "AS IS" BASIS, 
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and 
+    limitations under the License.
+-->
+
+## Carbondata 替换某商业列存DB查询性能对比
+
+本文主要在于给用户呈现Carbondata在替换某商业列存DB过程中对于该DB的查询性能提升，Carbondata自身的优势和特点，本文的数据仅为基于某领域查询特点框架下SQL的查询结果，只代表该特定查询特点下的性能对比。
+
+
+
+
+
+## 1.集群状态对比
+
+| 集群             | 描述                                                      |
+| ---------------- | --------------------------------------------------------- |
+| 某商业列存DB集群 | 1个加载节点，1个协调节点，1个查询节点，SSD硬盘            |
```

Comment:
please discribe how many CPU cores in both environements

Reply:
done